StreamingAnalytics/LimitedBandwidth/demo_publisher.py

```diff
@@ -4,40 +4,54 @@
 # Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG
 
 
-import paho.mqtt.client as mqtt
+import argparse
 import json
 import random
 import time
-import argparse
+import paho.mqtt.client as mqtt
 
 
 if __name__ == "__main__":
-	parser = argparse.ArgumentParser()
-	parser.add_argument('--host', dest='broker', type=str, default='localhost', help='The hostname where the MQTT broker is running.')
-	parser.add_argument('--port', dest='port', type=int, default=1883, help='The MQTT broker port to connect to.')
-	args = parser.parse_args()
-	
-	mqttClient = mqtt.Client("Demo-measurements")
-
-	def on_publish(client, userdata, result):
```

Comment:
Why are we now prefixing the arguments with underscores here?

Reply:
That’s conventions to ignore unused arguments (part of pylint rules)

Reply:
Okay.